src/test/run-make-fulldeps/coverage-spanview/expected_mir_dump.closure/closure.main-{closure#2}.-------.InstrumentCoverage.0.html

```diff
@@ -78,6 +78,7 @@
 100:12-100:20: @0[5]: _5 = (*((*_1).0: &amp;bool))">        let mut countdown = 0;</span></span>
 <span class="line"><span class="code even" style="--layer: 1" title="99:29-99:30: @0[1]: _3 = const 0_i32
 99:13-99:26: @0[2]: FakeRead(ForLet, _3)
+<<<<<<< HEAD
```

Comment:
Merge has broken here. Coverage tests probably need blessing since there have been other changes to these tests since #78373

Reply:
oops, sorry about that, thanks!

Reply:
(how did it pass CI with that in place? Do we not check this output as part of CI?)

Reply:
(Ah I bet I need to set `profiler = true` in my config.toml to get the relevant information here.)

Reply:
yes, this isn't run on the PR builders.

Reply:
Do you know if I need to do something *else* besides set `profiler = true`? I'm having trouble seeing the test failure in my local build, and thus I expect blessing here won't help until that's resolved.

Reply:
This is my entire config.toml
```toml
changelog-seen = 2

[llvm]

assertions = true

[build]

build = "x86_64-pc-windows-msvc"
profiler = true

[install]

[rust]

debug = true

[target.x86_64-unknown-linux-gnu]

[dist]
```
The test does take over a minute to run though. I can try on linux if that would help.

Reply:
Hmm. The profiler coverage tooling and file generation shouldn't be Windows specific, right? So weird that I don't see this.

I'll try to make my own setup more closely match yours, I guess.  (Also, @wesleywiser said they'd help; I'm going to get in touch with them soon.)

Reply:
@pnkfelix I checked out your PR locally and built with `./x.py test src/test/run-make-fulldeps/coverage`. The three coverage tests were ignored because I didn't have `profiler = true` set in my `config.toml`. After setting that, I reran the command, everything built and I got one failing test: `run-make-fulldeps/coverage-spanview`. I don't think I have anything special configuration other than `profiler = true` set.

I went ahead and blessed the test and pushed [here](https://github.com/rust-lang/rust/commit/6e95957ad7af56b66f39df54899aac44b75fc535) in case you just want to cherry-pick that.